### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by miamicityman** [Leantime/leantime#850] [General] [answered]
Hello there:

I'm either getting a 500 error, 400 error or a blank screen when trying to run at **subdomain.domain.com/install**.

I've installed leantime on the following directory:

- home/company/public_html/leantime/

My vhost domain is pointed to either:

- home/company/public_html/leantime/, or
- home/company/public_html/leantime/public/

What should my htaccess settings be at?

- public_html/leantime/public

I've tried numerous combinations of different routes without luck. 

Any help anyone? Thanks.

ps: Yes mod_rewrite is working.

**Comment by marcelfolaron** [comment, marked as answer]
Are you on PHP8? Leantime dropped support for php7 as of 2.2.0

**Reply by alkisal** [reply]
Updated to php 8.1 and all is fine. thank you, great job by the way!

**Comment by miamicityman** [comment]
Thank you 1000x. That was it.

**Comment by marcelfolaron** [comment]
No problem. Glad it works
